rdrf/rdrf/management/commands/create_translation_file.py

```diff
@@ -92,6 +92,11 @@ def _extract_html_strings(self, yaml_file):
             form_header_html = form_dict["header"]
             htmls.append(form_header_html)
 
+            # section headers
+            for section_dict in form_dict["sections"]:
+                section_header_html = section_dict["header"]
+                htmls.append(section_header_html)
+
```

Comment:
@ScriptSmith Is this correct? I've added this bit here, because the section header may (or most likely will) contain HTML. I noticed this is how we translate `splash_screen`, but then was confused because the form header gets translated here and then also gets yielded later on as if it was a standard string

Reply:
Looks like that needs the `--extract_html_strings` flag when the script is called, which currently isn't the case for angelman.

I'm not sure the flag even works, and its method of concatenating all the html doesn't seem great.

See if we can drop that method and move any functionality into the relevant methods that can treat html as just another string.